office-backend/exception/NoZaakFound.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+export class NoZaakFound extends Error {
```

Comment:
```suggestion
export class ZaakNotFound extends Error {
```

I think this is a more generally used naming convention and is also the one we use in ZAC.